lib/public/components/qcFlags/qcFlagsBreadcrumbs.js

```diff
@@ -9,13 +9,14 @@ import spinner from '../common/spinner.js';
  * Render QC flag(-s) breadcrumbs
  *
  * @param {object} qcBreacrumbsElements breadcrumbs elements
- * @param {Run} remoteRun run remote data
- * @param {number} [remoteDataPass] data pass remote data -- exclusive with `remoteSimulationPass`
- * @param {object} [remoteSimulationPass] simulation pass remote data -- exclusive with `remotedataPass`
- * @param {number} remoteDplDetector dpl detector remote data
+ * @param {Run} qcBreacrumbsElements.remoteRun run remote data
```

Comment:
Incorrect spelling of `qcBreadcrumbsElements` in all iterations below

Reply:
fixed